pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -1221,27 +1221,8 @@ func (lbaas *LbaasV2) EnsureLoadBalancerDeleted(clusterName string, service *v1.
 		if lbSecGroup.Err != nil && !isNotFound(lbSecGroup.Err) {
 			return lbSecGroup.Err
 		}
-
-		// Delete the rules in the Node Security Group
-		opts := rules.ListOpts{
-			SecGroupID:    lbaas.opts.NodeSecurityGroupID,
```

Comment:
The codes used to find rules of node security group and delete them, not for LB security group.
So the codes can't be deleted.